scripts/dependency-graph-generator/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.dot
```

Comment:
Do we know that the files will always land in this folder? 

For `*.dot` we could probably add that to the global `.gitignore`. (`*.png` would be harder since we do have some png files checked in.)

Reply:
yeah I've used __dirname to always set output to this folder

Reply:
I don't know if we always to want to add *.dot to gitignore for the repo or even if we will have more, will leave for now, it's not a big change